README.md

```diff
@@ -77,13 +76,18 @@ dune build @doc
   for the general one)
 
 - Since our Patricia Tree use big-endian order on keys, the maps and sets are
-  sorted in increasing order of keys. We only support positive integer keys.
+  sorted in increasing **unsigned order** of keys.
+  This means negative keys are sorted above positive keys, with `-1` being the
+  largest possible key, and `0` the smallest.
   This also avoids a bug in Okasaki's paper discussed in [*QuickChecking Patricia Trees*](https://www.cs.tufts.edu/comp/150FP/archive/jan-midtgaard/qc-patricia.pdf)
   by Jan Mitgaard.
+
+  It also affects functions like `unsigned_min_binding` and `pop_unsigned_minimum`. They will return the smallest
+  positive integer of both positive and negative keys are present; and not the smallest negative, as one might expect.
 - Supports generic maps and sets: a `'m map` that maps `'k key` to `('k, 'm) value`.
   This is especially useful when using [GADTs](https://v2.ocaml.org/manual/gadts-tutorial.html) for the type of keys. This is also sometimes called a dependent map.
 - Allows easy and fast operations across different types of maps and set (e.g.
-  an intersection between a map and a set), since all sets and maps, no matter their key type, are really positive integer sets or maps.
+  an intersection between a map and a set), since all sets and maps, no matter their key type, are really integer sets or maps.
```

Comment:
I am not sure what this means (even before the change)

Reply:
Yes that wasn't very clear. I rephrased it.